src/main/java/org/scijava/script/AbstractScriptLanguage.java

```diff
@@ -70,6 +73,11 @@ public String getLanguageName() {
 		if (info != null) name = info.getName();
 		return name != null && !name.isEmpty() ? name : inferNameFromClassName();
 	}
+    
+    @Override
+    public AutoCompleter getAutoCompleter(ScriptEngine scriptEngine) {
+        return new DefaultAutoCompleter(this, scriptEngine);
```

Comment:
Now that we have Java 8, this could be a `default` method of `ScriptLanguage` interface itself, rather than in the abstract base class. That would potentially be "more backwards compatible", just in case anyone out there has implemented the `ScriptLanguage` interface directly.